test/test_po_files.py

```diff
@@ -8,7 +8,7 @@
 from . import fixture
 import six
 
-from subscription_manager import managercli
+from subscription_manager.cli_command.cli import system_exit
```

Comment:
In other parts of the code 'system_exit' was imported from subscription_manager.cli - why is it being imported from subscription_manager.cli_command.cli here?